src/lib/skilling/skills/hunter/defaultBirdHouseTrap.ts

```diff
@@ -0,0 +1,16 @@
+export interface BirdhouseData {
+	lastPlaced: string | null;
+	birdhousePlaced: boolean; // false -> nothing placed, true -> something placed
```

Comment:
this isn't document properly, make an empty line above the birdhousePlaced line, then type `/**` and enter, put info in there instead.